modules/openapi-generator-api/pom.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>openapi-generator-project</artifactId>
+    <groupId>org.openapitools</groupId>
+    <version>3.2.2-SNAPSHOT</version>
+    <relativePath>../..</relativePath>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>openapi-generator-api</artifactId>
```

Comment:
Can this package be renamed to `openapi-generator-core`? The `-api` seems redundant with the `openapi`, and some may confuse this package with `openapi-generator-online`.

Reply:
yep, that's a better name for sure.